bin/kano-stop-splash

```diff
@@ -15,5 +16,6 @@ if test -n "$SPLASH_PID" ; then
 	   kill -ALRM $SPLASH_PID
        fi
    fi
+else
+    pkill -f kano-start-splash
 fi
```

Comment:
Is there really no way to pass SPLASH_PID in? I was hoping to use kano-start-splash to do the popups for the screen capture tool. It would be better if kano-stop-splash didn't randomly kill other popups.

Reply:
Oh, totally agreed @Ealdwulf , It's not so easy because it happens from separate parts on the OS but I'll think of a solution.